src/HospitalRun.tsx

```diff
@@ -42,6 +43,15 @@ const HospitalRun = () => {
                   path="/patients/new"
                   component={NewPatient}
                 />
+                <PrivateRoute
+                  isAuthenticated={
+                    permissions.includes(Permissions.WritePatients) &&
+                    permissions.includes(Permissions.ReadPatients)
+                  }
+                  exact
+                  path="/patients/edit/:id"
+                  component={EditPatient}
+                />
```

Comment:
I think that we need some tests in `HospitalRun.test.tsx` surrounding this new route, especially for permissions.

Reply:
Added these